rep-0144.rst

```diff
@@ -102,7 +102,7 @@ Special Suffixes:
   ``pr2_robot``
 * a package containing the URDF and meshes of a robot should be named 
   ``<name_of_the_robot>_description``, e.g ``pr2_description``
-* if a package is meant for test only, prepend ``test``
+* if a package is meant for test only, append ``tests``
```

Comment:
there are some (less I agree) that have a plural. I changed to plural here for consistency with `plugins` and `msgs`. @dirk-thomas , we probably need some English speaker to tell us what their feel is here :)